pallets/claims/src/mock.rs

```diff
@@ -156,13 +166,13 @@ impl pallet_balances::Config for MockRuntime {
     type MaxLocks = ();
 }
 
-// Parameterize RAD claims pallet
+// Parameterize claims pallet
 parameter_types! {
-    pub const RadClaimsPalletId: PalletId = PalletId(*b"rd/claim");
+    pub const ClaimsPalletId: PalletId = PalletId(*b"p/claims");
     pub const One: u64 = 1;
     pub const Longevity: u32 = 64;
     pub const UnsignedPriority: TransactionPriority = TransactionPriority::max_value();
-    pub const MinimalPayoutAmount: node_primitives::Balance = 5 * currency::RAD;
+    pub const MinimalPayoutAmount: node_primitives::Balance = 5 * RAD;
```

Comment:
This `RAD` variable should be renamed to `CFG` and all the sub-precision ones. This was done in the master branch already, I guess we branch out before that change went in.

Reply:
Okay. Done. Such dependency from a pallet to a runtime should be avoided, so that to be able to build the runtime "by composition" of pallets (i.e. independent business logic entities). Why not having a library with common constants, traits and functions, like "centrifuge-support::traits, centrifuge-support::primitives, ..."?